packages/@webex/plugin-cc/src/services/core/Utils.ts

```diff
@@ -10,6 +10,12 @@ const getCommonErrorDetails = (errObj: WebexRequestPayload) => {
   };
 };
 
+export const getErrorReason = (error: any): string => {
```

Comment:
Decided to write a util method, it can be reused also.

Reply:
we have this already right? getErrorDetails can enhance or reuse the same